README.md

```diff
@@ -54,15 +58,15 @@ let g:neomake_javascript_enabled_makers = ['jshint']
 For use with the `:Neomake` command (makers that run on an individual file), it
 is critical that makers follow this naming convention:
 
-    g:neomake_{ language }_{ makername }_maker
+    g:neomake_{ filetype }_{ makername }_maker
 
-Where `{ language }` is replaced with the name of the language, and
+Where `{ filetype }` is replaced with the name of the filetype, and
 `{ makername }` is replaced with the name that you want your maker to have. If
 your maker does not follow this convention, Neomake will not be able to see
 it, and you will get an error message like `{ makername } not found`.
 
 Explanation for the strings making up the errorformat can be found by typing
-`:h errorformat` in Neovim (or Vim).
+`:h errorformat` in [Neovim] (or Vim).
```

Comment:
This is rendering an invalid link, I would say to remove the last part, keeping it as:

Explanation for the strings making up the errorformat can be found in `:h erroformat`.